docs/examples/container-config.json

```diff
@@ -3,7 +3,7 @@
     "name": "busybox"
   },
   "image":{
-    "image": "busybox"
+    "image": "index.docker.io/busybox"
```

Comment:
The `index` subdomain is not required from my point of view: 
```suggestion
    "image": "docker.io/busybox"
```

Reply:
@vorburger could you remove the `index` prefix?

Reply:
@feiskyer done. (Sorry it took a moment.) Merge?

Reply:
It looks like it's still there :)